From 81f0a50ff4340a0533e4a47c9e68e8c9fcd41dde Mon Sep 17 00:00:00 2001
From: Andrey <alevoshenya@gmail.com>
Date: Mon, 1 May 2023 18:34:25 +0300
Subject: [PATCH] Changed 2 lines in firstFile.txt in git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 027870f..be8b3ba 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-line0
-line1
+line0 seven
+line1 seven
 line2
 line3
 line4
-- 
2.39.1.windows.1

